packages/back-end/src/routers/users/users.controller.ts

```diff
@@ -229,11 +229,15 @@ export async function getRecommendedOrgs(req: AuthRequest, res: Response) {
         const currentUserIsPending = !!org?.pendingMembers?.find(
           (m) => m.id === user.id
         );
+        const currentUserIsMember = !!org?.members?.find(
```

Comment:
I could change this to just filter out members from the result here, then no need to do it client side.  It kind of makes sense that we wouldn't recommend an org that they are already a member of.  It's possible though the final design we want to show both the orgs that they are currently members of and the orgs that they can join, in which case we will need it like this.